src/main/java/view/InputView.java

```diff
@@ -0,0 +1,94 @@
+package view;
+
+import domain.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static domain.PlayerCollection.createPlayerCollection;
+import static domain.PrizeCollection.createPrizeCollection;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static List<Player> players = new ArrayList<>();
+    private static final int PLAYERS_MAX_NAME_LENGTH = 5;
+    private static final int LADDERS_MIN_HEIGHT = 1;
+
+    public static String inputPlayerName() {
+        System.out.println("참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)");
+        String playerName = scanner.nextLine();
+
+        players = createPlayerCollection(playerName);
+
+        validatePlayerNameUnderFive(players);
+
+        return playerName;
+    }
```

Comment:
inputView의 역할은 무엇인가요?
현재 이 코드에는 inputView의 역할이 
1) 글자 입력받기,
2) 구분자 기준으로 글자 나누기,
3) 5글자 아래로 글자 수 제한하기
이렇게 3가지의 역할을 하는 것으로 보여집니다.
inputView는 입력의 역할을 수행하는 담당이므로 입력의 역할만 담당하는 것이 좋다고 생각합니다.

Reply:
아래와 같이 하나의 함수가 하나의 일을 처리하도록 바꿔보는것이 어떨까요?
```suggestion
public static String inputPlayerName() {
        System.out.println("참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)");
        return scanner.nextLine();
    }
```

Reply:
이 부분에서 궁금한 점이 하나 있어요! "1메서드 1기능" 이라는 키워드에서 저는 "기능"을 로직(?)으로 생각을 했는데요.  위 클래스의 경우 "기능 == 스캐너 입력" 이라고 생각했습니다. 그 후에 있는 예외처리는 "기능"이라고 생각하지 않았거든요.. 그렇다면 예외처리는 언제 하는 것이 가장 바람직한 지 궁금합니다!(프리코스를 진행하면서도 이에 대해 의문이 생기더라구요..)

1. 메서드의 예외처리는 해당 메서드 안에서 실행(예외 코드 구현은 메서드 외부에서!)
2. Controller -> Application(실행만 하면 됨) 의 순서(구조)로 프로젝트를 진행하게 된다면 Controller에서 예외처리하기

저는 이 두가지가 생각이 났는데 이 중에 뭐가 더 바람직한 지 아직 잘 모르겠습니다...ㅜ 혜빈님은 어떻게 하시나요..?

Reply:
예외처리는 가장 밀접한 관계를 갖는 객체에 넣는 것이 좋다고 생각합니다.

예를 들어 
입력값은 정수여야한다. <- inputView 객체의 예외처리
입력값이 1~45 사이의 수여야 한다. <- lotto객체의 예외처리

이런식으로 관련된 객체에 넣는 것이 적합하다고 생각합니다!